openstudiocore/src/qtwinmigrate/qwinwidget.cpp

```diff
@@ -56,6 +56,10 @@
 
 #include <qevent.h>
 
+// DLM: not sure how to do this right, think I only need this for static builds
```

Comment:
@kbenne this probably needs to be inside an #if defined STATIC_BUILD or something in the case we are linking with dynamic Qt, not sure what we are going with